### PR TITLE
tests: Fix putWithLease traffic timeout/limit and double overall timeout to reduce number of failed requests

### PR DESCRIPTION
**Pull request by serathius**
Maybe due addition of leases, but I noticed increases number of request timeout even without any failure injections. 
cc @ahrtr @ptabor @geetasg

### CODE REVIEW

**Comment by serathius** [comment]
cc @geetasg

**Comment by serathius** [comment]
Before
image

**Comment by serathius** [comment]
After 
image

File: tests/linearizability/traffic.go [hunk 1]
```diff
@@ -75,33 +76,38 @@ func (t readWriteSingleKey) Read(ctx context.Context, c *recordingClient, limite
...
 	case PutWithLease:
 		leaseId := lm.LeaseId(cid)
 		if leaseId == 0 {
-			leaseId, err = c.LeaseGrant(ctx, t.leaseTTL)
-			lm.AddLeaseId(cid, leaseId)
+			leaseId, err = c.LeaseGrant(writeCtx, t.leaseTTL)
+			if err != nil {
+				lm.AddLeaseId(cid, leaseId)
+				limiter.Wait(ctx)
+			}
```
Comment (ahrtr) [inline tests/linearizability/traffic.go:100]: Do you intentionally ignore the lease when `err == nil`? 

```suggestion
			lm.AddLeaseId(cid, leaseId)
			if err != nil {
				limiter.Wait(ctx)
			}
```

**Comment by ahrtr** [comment]
It would be better to support limiting the number of lease in test case, and print more stats info (e.g. number of lease) for lease.

File: tests/linearizability/traffic.go (hunk 1 above)
Comment (serathius) [reply tests/linearizability/traffic.go]: Ups, I had wrong condition.

**Comment by serathius** [comment]
> It would be better to support limiting the number of lease in test case, and print more stats info (e.g. number of lease) for lease.

Don't understand. Leases are limited to 1 per client.

**Review by ptabor — APPROVED** [review]

**Comment by geetasg** [comment]
Thanks Marek.

**Pull request merged**